src/Echidna.Tests/Utilities/TestHelper.cs

```diff
@@ -0,0 +1,14 @@
+namespace Medallion.Data.Tests;
+
+internal static class TestHelper
+{
+    public static bool IsOnCIPipeline => bool.TryParse(Environment.GetEnvironmentVariable("CI"), out var parsed) && parsed;
+
+    public static IEnumerable<Db> DbsToTest()
+    {
+        var allDbs = Enum.GetValues(typeof(Db)).OfType<Db>();
+        return IsOnCIPipeline
+            ? allDbs.Except(new[] { Db.Oracle, Db.MariaDb })
```

Comment:
@madelson Oracle seems to take up a lot of time/space to install, so I'm not sure if it's worth setting up in the CI pipeline.